Common/Data/FileSync.swift

```diff
@@ -103,51 +120,42 @@ public class FileSync: NSObject {
                 }
 
                 return Future(value: files)
-            }
+            }.onSuccess { completionBlock($0, nil) }.onFailure { completionBlock(nil, $0) }
```

Comment:
Please multiple lines for this.

Alternatively, see first comment and use `.onComplete { completionBlock($0) }